sdk/storage/inc/blobs/page_blob_client.hpp

```diff
@@ -98,6 +98,16 @@ namespace Azure { namespace Storage { namespace Blobs {
      */
     PageBlobClient WithSnapshot(const std::string& snapshot) const;
 
+    /**
+     * @brief Creates a clone of this instance that references a version ID rather than the base
+     * blob.
+     *
+     * @param versionId The version ID returning a URL to the base blob.
+     * @return A new PageBlobClient instance.
+     * @remarks Pass empty string to remove the version ID returning the base blob.
+     */
+    PageBlobClient WithVersionId(const std::string& versionId) const;
```

Comment:
`VersionID`? Two character acronym should be all caps.
ditto in other places.

Reply:
This is not an acronym

Reply:
It's an abbreviation. @BillyONeal Do rules apply to acronym also apply to abbreviation?

Reply:
it says `Acronyms and initialisms`. https://azure.github.io/azure-sdk/cpp_design.html#definitions